googlepay/src/main/java/com/airwallex/android/googlepay/Constants.kt

```diff
@@ -0,0 +1,17 @@
+package com.airwallex.android.googlepay
+
+import com.google.android.gms.wallet.WalletConstants
+
+object Constants {
+    val DEFAULT_SUPPORTED_METHODS = listOf(
+        "PAN_ONLY",
+        "CRYPTOGRAM_3DS"
+    )
+
+    val DEFAULT_SUPPORTED_CARD_NETWORKS = listOf(
+        "MASTERCARD",
+        "VISA"
```

Comment:
Do we need UnionPay and Meastro?

Reply:
No, according to @shirly-chen-awx , we're not supporting these two in googlepay